rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] This should be within the "attr_accessor" and not the "attr_reader" thus making it something like this : ```attr_accessor :date, :book, :person```

Reply:
Done! Thanks